qutebrowser/utils/log.py

```diff
@@ -505,7 +505,12 @@ def dump_log(self, html=False, level='vdebug'):
             if record.levelno >= minlevel:
                 lines.append(fmt(record))
         return '\n'.join(lines)
-
+        
```

Comment:
It seems your editor has added (invisible) whitespace at the end of line here, and in various other places (see the [pylint log](https://travis-ci.org/The-Compiler/qutebrowser/jobs/149658740#L492-L506). What editor are you using? Can you somehow configure it to automatically strip such whitespace?